.github/workflows/ateam.yml

```diff
@@ -19,6 +19,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Create yum repo
+        run: |
+          sudo dnf install -y createrepo_c python3-pip
+          pip3 install -U pip
+          pip3 install -r ./ci/create_yum_repo/requirements.txt
+          ./ci/create_yum_repo/main.py
```

Comment:
This looks much better :smile: 
Be aware that all this is running inside the container, which means that the repo is created inside the container. The next step (**Image build**) will run in the bare-metal machine, so it won't see this container or the repo just created here.
For the **Image Builder** (`osbuild`) to see the repo you'll need to do one of two things:
- Rsync the repo directory to the bare-metal machine (`${{ secrets.HOST }}`)
- Upload to S3 and use it from there.

I like the second option more.
Here is an axample from `osbuild`: https://github.com/osbuild/osbuild-composer/blob/main/schutzbot/mockbuild.sh#L111
Here is also an article about it. In case it helps.

Reply:
Thanks. Pushed the changes.